README.md

```diff
@@ -15,6 +15,28 @@ Revisit your solution to homework #1, problem 5, which expresses an improper fra
 **a)** Write a function `fractionString(n,d)` which takes 2 parameters (n,d) and returns a string.  For example, 
 `fractionString(7,4)` should return "1 3/4", and `fractionString(3,3)` should (for now) return "1 0/3".  As before, assume that _n_ and _d_ are both positive integers.
 
+
+```
+//Question 1
+
+//a & b
+
+function fractionString(n, d){
+  var rem = n % d;
+  if (d === 0) {
+  return n;
+  } else if (n > d && rem == 0) {
+    return (Math.floor(n/d));
+    } else if (n > d) {
+    return (Math.floor(n/d) + " " + rem + "/" + d);
+    }  else if (n == d) {
+      return(n);
+    } else {  
+         return (n + "/" + d);
+  }
+};
```

Comment:
1) You have a few edge cases here that do some funny things. 4/0 should probably not return 4, for example. `infinity` might be the accepted answer, or perhaps `NaN`--I'd have to look that up. And 4/4 should of course return 1; at present it returns 4.